project/StreamOperatorsIndexGenerator.scala

```diff
@@ -140,8 +147,17 @@ object StreamOperatorsIndexGenerator extends AutoPlugin {
         "akka-stream/src/main/scala/akka/stream/javadsl/StreamConverters.scala",
         "akka-stream/src/main/scala/akka/stream/scaladsl/FileIO.scala",
         "akka-stream/src/main/scala/akka/stream/javadsl/FileIO.scala",
+
+        // akka-stream-typed
+        "akka-stream-typed/src/main/scala/akka/stream/typed/javadsl/ActorSource.scala",
+        "akka-stream-typed/src/main/scala/akka/stream/typed/scaladsl/ActorSource.scala",
+        "akka-stream-typed/src/main/scala/akka/stream/typed/javadsl/ActorFlow.scala",
+        "akka-stream-typed/src/main/scala/akka/stream/typed/scaladsl/ActorFlow.scala",
+        "akka-stream-typed/src/main/scala/akka/stream/typed/scaladsl/ActorSink.scala",
+        "akka-stream-typed/src/main/scala/akka/stream/typed/javadsl/ActorSink.scala",
       ).flatMap{ f ⇒
-        val element = f.split("/")(7).split("\\.")(0)
+        val slashesNr = f.count(_ == '/')
+        val element = f.split("/")(slashesNr).split("\\.")(0)
```

Comment:
FYI @raboof this is to make it dynamically work when things are nested deeper

Reply:
could maybe even be `f.split("/").last.split("\\.").head` then? but :+1: anyway